src/components/AssistantTextProcessingModal.vue

```diff
@@ -6,7 +6,7 @@
 	<NcModal v-if="show"
 		:size="modalSize"
 		:can-close="false"
-		:name="t('assistant', 'Nextcloud Assistant')"
+		:aria-label="t('assistant', 'Nextcloud Assistant')"
```

Comment:
You do not have to wait for the fix you can already force it:

```suggestion
		dark
		:name="t('assistant', 'Nextcloud Assistant')"
```

Reply:
Good to know, thanks!